### ISSUE TITLE
CI is broken

### ISSUE THREAD

**Issue by maxd** [cequel/cequel#266]
Seems like CI is broken. [Look at tests](https://travis-ci.org/cequel/cequel/builds/98101630) for my PR #265:

```
...
Installing ione 1.2.3
Installing cassandra-driver 2.1.5
NoMethodError: undefined method `spec' for nil:NilClass
An error occurred while installing
cequel (1.7.0), and Bundler cannot
continue.
Make sure that `gem install cequel -v
'1.7.0'` succeeds before bundling.
The command "eval bundle install --without=debug" failed. Retrying, 2 of 3.
...
```

**Comment by maxd** [comment]
Can anybody to repair CI?

**Comment by pezra** [comment]
fixed by #289
